pkg/scd/dss_report_handler.go

```diff
@@ -0,0 +1,77 @@
+package scd
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/google/uuid"
+	"github.com/interuss/dss/pkg/api"
+	restapi "github.com/interuss/dss/pkg/api/scdv1"
+	dsserr "github.com/interuss/dss/pkg/errors"
+	"github.com/interuss/dss/pkg/logging"
+	"github.com/interuss/stacktrace"
+	"go.uber.org/zap"
+)
+
+// DSSReporting takes care of handling a DSS report.
+type DSSReporting interface {
```

Comment:
I was considering this but avoided it, because we already have some files called `*_handlers` that seem to be specifically about handling REST calls?

This interfaces is about handling things _after_ they traversed the `MakeDssReport` function in `dss_report_handler` and it feels a bit weird to pass that to an interface called DSSReportHandler?

(Having things in a separate package would make it clearer: I'm open to suggestions)

Reply:
True that. That's a reason why I'm not a fan of defining interfaces ahead of time without it being actually in use TBH ([c.f. as well this](https://go.dev/wiki/CodeReviewComments#interfaces)). But that's (still) nitpicking so I'm good merging it as it is. I leave it up to you.

Reply:
Thanks for the pointer. I'm still thinking too much in terms of Java interfaces -> i did some renaming to get closer to the spec.